src/adBreak.js

```diff
@@ -38,6 +38,17 @@ function start(player) {
   // This removes the native poster so the ads don't show the content
   // poster if content element is reused for ad playback.
   player.ads.removeNativePoster();
+
+  // Ensure ads are watched at x1 speed and speed cannot be changed for the duration of the ad
+  player.ads.preAdPlaybackRate_ = player.playbackRate();
+  player.playbackRate(1);
+  if (player.controlBar.playbackRateMenuButton.playbackRateSupported &&
```

Comment:
Will `player.control.playbackRateMenuButton` always be initialized here? If so, no change is needed

Reply:
It's definitely worth doing null checks for the entire chain.

Reply:
Thanks. I'll add the null checks.